src/js/ProductDetails.mjs

```diff
@@ -36,7 +36,9 @@ export default class ProductDetails {
       .addEventListener("click", this.addToCart.bind(this));
   }
   addToCart() {
-    setLocalStorage("so-cart", this.product);
+    const cart = getLocalStorage("so-cart") || [];
+    cart.push(this.product);
+    setLocalStorage("so-cart", cart);
```

Comment:
No not forget to include your previous bug fixes in new code. This is from the Week 1 cart fix.